### DISCUSSION TITLE
https://steinkamp.us/posts/2024-08-26_the-story-of-quantize-midi

### DISCUSSION THREAD

**Discussion by giscus** [zsteinkamp/steinkamp.us#15] [Announcements]
# https://steinkamp.us/posts/2024-08-26_the-story-of-quantize-midi

All about the Max for Live device that can variably quantize a MIDI note stream in real time.

https://steinkamp.us/posts/2024-08-26_the-story-of-quantize-midi

**Comment by bgsulz** [comment]
I love this one!

**Reply by zsteinkamp** [reply]
Thanks @bgsulz! I'm happy to hear any ideas for improvements, or other device ideas you may have.
